### PR TITLE
Db 252 cross file

### PR DESCRIPTION
**Pull request by jworcestBAH**
Cross file validation, now including row numbers in error reports

### CODE REVIEW

File: dataactvalidator/validation_handlers/validator.py [hunk 1]
```diff
@@ -72,6 +72,7 @@ def evaluateCrossFileRule(cls, rule, submissionId, record = None):
             fieldsToCheck = cls.cleanSplit(rule.rule_text_1,True)
             # For each entry, check for the presence of matching values in second table
             for thisRecord in sourceRecords:
+                print("thisRecord: " + str(type(thisRecord)) + ", " + str(thisRecord))
```
Comment (nmonga91) [inline dataactvalidator/validation_handlers/validator.py]: @jworcestBAH Are the print statements in this file necessary? If not, they should be removed.

File: tests/baseTestValidator.py [hunk 2]
```diff
@@ -86,7 +86,7 @@ def tearDownClass(cls):
         cls.interfaces.close()
         dropDatabase(cls.interfaces.jobDb.dbName)
         dropDatabase(cls.interfaces.errorDb.dbName)
-        dropDatabase(cls.interfaces.stagingDb.dbName)
```
Comment (nmonga91) [inline tests/baseTestValidator.py]: @jworcestBAH Is there a reason this is commented out whereas it was uncommented previously?

Comment (jworcestBAH) [inline tests/baseTestValidator.py]: Good catch, uncommented this.

**Comment by jworcestBAH** [comment]
Removed extra print statements

**Comment by nmonga91** [comment]
Tests pass and error report looks good.

**Pull request merged**